modules/dicom_archive/php/dicom_archive.class.inc

```diff
@@ -48,6 +48,9 @@ class Dicom_Archive extends \NDB_Menu_Filter
      */
     function _setupVariables()
     {
+        // Rest the filter because this is broken with react modules
+        $this->_resetFilters();
```

Comment:
Makes sense. I forgot that dicom archive was one where filtering is done front-end side.